scripts/apps/web-publisher/styles/_helpers_variables.sass

```diff
@@ -0,0 +1,45 @@
+$superdesk-green:       rgba(49,172,102,1);
```

Comment:
branding is the same but for some reason I can't use global variables defined in variables.scss so I just copied them. There are/will be new elements that might be usefull for other components in superdesk. We can talk about putting them in main styles folder later.